gapis/resolve/dependencygraph2/graph_visualization/graph_visualization.go

```diff
@@ -0,0 +1,21 @@
+package graph_visualization
```

Comment:
Add copyright notice to all the new files. You can just copy it from one of the other go files, and make sure the year says 2018.

Reply:
I added it!! :+1: